src/GitHub.VisualStudio/GitHubPackage.cs

```diff
@@ -25,7 +26,7 @@ namespace GitHub.VisualStudio
     [ProvideAutoLoad("11B8E6D7-C08B-4385-B321-321078CDD1F8")]
     [ProvideToolWindow(typeof(GitHubPane), Orientation = ToolWindowOrientation.Right, Style = VsDockStyle.Tabbed, Window = EnvDTE.Constants.vsWindowKindSolutionExplorer)]
     [ProvideOptionPage(typeof(OptionsPage), "GitHub for Visual Studio", "General", 0, 0, supportsAutomation: true)]
-    public class GitHubPackage : Package
+    public class GitHubPackage : AsyncPackage
```

Comment:
Update your `PackageRegistration` attribute to include `AllowsBackgroundLoading = true`